CHANGELOG

```diff
@@ -111,6 +111,9 @@ Miscellaneous
 - Added support for output in the ParaView XML format. See Examples 5/5p, 9/9p
   and the new ParaViewDataCollection class.
 
+- Added test to prevent the unintentional introduction of a binary file in the
+  repo.
+
```

Comment:
Needs to be updated to document the other tests in `tests/scripts` and to also mention the impact on Travis